slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt

```diff
@@ -172,6 +172,16 @@ class SlideToActView
                 }
             }
 
+        /**
+         * Retrieves the current position of the slider cursor.
+         *
+         * @return The current position of the slider cursor as an integer.
+         * @see getSliderCursorPosition
+         */
+        fun getSliderCursorPosition(): Int {
+            return mPosition
+        }
```

Comment:
I think we should instead rename `mPosition` to `sliderPosition` and make it `public` rather than creating a custom getter